airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -679,14 +680,9 @@ async def _load_config(self):
             async_config.load_incluster_config()
             return async_client.ApiClient()
 
-        if kubeconfig_path:
-            self.log.debug(LOADING_KUBE_CONFIG_FILE_RESOURCE.format("kube_config"))
-            self._is_in_cluster = False
```

Comment:
@VladaZakharova shall we specify this variable as `False` in the new implementation?